oak/server/rust/oak_runtime/src/proxy.rs

```diff
@@ -67,6 +68,7 @@ impl RuntimeProxy {
             node_infos: RwLock::new(HashMap::new()),
             next_node_id: AtomicU64::new(0),
             aux_servers: Mutex::new(Vec::new()),
+            introspection_event_queue,
```

Comment:
nit: might as well be consistent with the other fields and have `introspection_event_queue: Mutex::new(VecDeque::new()),` inline.